### ISSUE TITLE
Setting correct kwargs for jax models when in training mode

### ISSUE THREAD

**Issue by ndrakulicTT** [tenstorrent/tt-xla#1388]
### Is your feature request related to a problem? Please describe

Right now we need to set manually kwargs for each model when in training mode. In most jax models this is not done properly (e.x. batch_norm and dropout).
Needs further investigation to find needed set of args.
